Resources/Prototypes/_NF/Shipyard/crossroads.yml

```
@@ -0,0 +1,40 @@
+# Author Info
+# GitHub: CadenceArgent (https://github.com/CadenceArgent)
+# Discord: Cadence
+
+# Shuttle Notes:
+# Docking Hub Vessel, with basic service setup, reception desk, and a public garden.
+- type: vessel
+  id: Crossroads
+  parent: BaseVessel
+  name: PHI Crossroads
+  description: "A spacious and inviting service ship. The crossroads is designed to help with dock crowding, connect communities, and serve as anchors for gatherings."
+  price: 85000 #appraises at ~70k with the appraisegrid command, marking up at 20% for upper-mid T2
+  category: Medium
+  group: Shipyard
+  shuttlePath: /Maps/_NF/Shuttles/crossroads.yml
+  guidebookPage: ShipyardCrossroads
+  class:
+  - Civilian
+  engine:
+  - Uranium
+
+- type: gameMap
+  id: Crossroads
+  mapName: 'PHI Crossroads'
+  mapPath: /Maps/_NF/Shuttles/crossroads.yml
+  minPlayers: 0
+  stations:
+    Crossroads:
+      stationProto: StandardFrontierVessel
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: 'Crossroads {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: '14'
+        - type: StationJobs
+          availableJobs:
+            Contractor: [ 0, 0 ]
+            Pilot: [ 0, 0 ]
+            Mercenary: [ 0, 0 ]
```

```suggestion
            ContractorInterview: [ 0, 0 ]
            PilotInterview: [ 0, 0 ]
            MercenaryInterview: [ 0, 0 ]
```
Use new interview jobs.

---

This will be fixed in the next commit, thanks for letting me know!